.github/workflows/get-go-version.yml

```diff
@@ -0,0 +1,24 @@
+name: Reusable Go Version Workflow
```

Comment:
Had to add this in here as it's a public repo so does not have access to the shared actions repository.

Reply:
Perhaps that actions repo should be public then?

Reply:
I'd prefer not mostly because of the *other* workflows that exist.